src/modules/admin/admin-tab-categories/AdminTabCategories.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+import { NavLink, useRouteMatch, useHistory } from 'react-router-dom';
+
+import { Button as MuiButton } from '@material-ui/core';
+import UnfoldMoreIcon from '@material-ui/icons/UnfoldMore';
+
+import { Button } from 'ui';
+
+import { Url } from 'utils';
+
+import csx from './AdminTabCategories.scss';
+
+export enum AdminTabCategory {
+  TECHNOLOGIES = 'technologies',
+  PATTERNS = 'patterns'
+}
+
+const ADMIN_TAB_CATEGORIES: AdminTabCategory[] = [
+  AdminTabCategory.TECHNOLOGIES,
+  AdminTabCategory.PATTERNS
+];
+
+const AdminTabCategories = () => {
+  const { location, push } = useHistory();
+
+  const {
+    params: { category: activeCategory }
+  } = useRouteMatch<{ category: AdminTabCategory }>();
```

Comment:
You should add model for this {category: AdminTabCategory}  = interface AdminRouteProps { category: AdminTabCategory } (inside newly created models.ts) file